playground/app.js

```diff
@@ -288,21 +260,30 @@ async function buildVueApp() {
           groupname: 'Group 1',
         },
       });
-      setupInitialData(this.$store);
+      const collections = await mongoservice.listCollections();
+      store.commit(VQBnamespace('setDomains'), { domains: collections });
+      store.dispatch(VQBnamespace('updateDataset'));
     },
     computed: {
-      code: function() {
+      activePipeline: function(){
         let activePipeline = this.$store.getters[VQBnamespace('activePipeline')];
         if (!activePipeline) {
           return '';
```

Comment:
I think the expected type of this.activePipeline is not a string but a pipeline object.
Better to return `undefined ` or `null` here, and let the computed below stringify this result.